File: src/main/java/com/clearcont/clearcontapp/repository/ControleRepository.java
package com.clearcont.clearcontapp.repository;

import com.clearcont.clearcontapp.model.Controle;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ControleRepository extends JpaRepository<Controle, Long> {
}

File: src/main/kotlin/br/com/clearcont/clearcontwebapp/repositories/ControleRepository.kt
package br.com.clearcont.clearcontwebapp.repositories

import br.com.clearcont.clearcontwebapp.models.Controle
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ControleRepository : JpaRepository<Controle, Long>
